app/Forms/UpdateProfileForm.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace App\Forms;
+
+use Kris\LaravelFormBuilder\Form;
+use Carbon\Carbon;
+use App\Member;
+use App\College;
+use App\Course;
+use App\Scholarship;
+use Illuminate\Http\Request;
+use Illuminate\Routing\Controller;
+use Auth;
+use DB;
+
+class UpdateProfileForm extends Form
+{
+	private $details = null;
+	public function __construct()
+	{
+		$user = Auth::user();
+		$this->details = DB::table('members')->where('crsid', $user->crsid)->first();
+	}
+	
+    public function buildForm()
+    {
+        $this
+            ->addBio()
+            ->add('date_of_birth', 'date', ['default_value' => $this->details->date_of_birth])
```

Comment:
That's actually a good catch: I think it was because I was modelling it after the existing forms and the fresher signup form did not request DoB (TPL said that it was sensitive), so in [`FreshersSignupForm`](https://github.com/CUMSA/members/blob/3027208ca46c52a1e78f72c1d4f344eee55bada4/app/Forms/FreshersSignupForm.php) DoB is not requested. `UpdateProfile` is currently copied from MembersSignUpForm instead of extending the class.